### PR TITLE
GTPs and GWP500 from AR6

### PR DESCRIPTION
**Pull request by znicholls**

### CODE REVIEW

File: globalwarmingpotentials.csv [hunk 1]
```diff
@@ -6,94 +6,97 @@
...
+HFE347mcc3,,575,530,641,576,2020,164,134,106
+HFE347pcf2,,580,889,1072,980,3370,279,237,181
+HFE356pcc3,,110,413,500,277,995,79,62.1,50.4
+HFE449s1,,297,421,509,460,1620,131,106,84.6
+HFE569sf2,,59,57,69,60.7,219,17.3,13.4,11
+HFE4310pccc124,,1870,2820,3353,3220,8720,920,1560,647
+HFE236ca12,,2800,5350,6260,6060,11700,1770,5020,1860
+HFE338pcc13,,1500,2910,3466,3320,9180,948,1520,657
+HFE227ea,,,6450,7377,7520,9800,2570,7720,4440
+HCFE236ea2,,,1790,2143,2590,7020,741,1260,521
```
Comment (znicholls) [inline globalwarmingpotentials.csv:86]: Was this a typo previously? I couldn't find HFE236ea2 in AR6...

**Comment by znicholls** [comment]
I did this by hand while bored so would be good if someone who has a csv and an automated process can check

File: globalwarmingpotentials.csv (hunk 1 above)
Comment (rgieseke) [reply globalwarmingpotentials.csv:86]: Seems likely! @jkikstra can you confirm/check with your CSV/Excel?

Comment (jkikstra) [reply globalwarmingpotentials.csv:86]: Might be a typo indeed.
But not sure which is better to use.

HCFE-236ea2 is in AR6, both in the PDF and in the csv from @chrisroadmap.

Google tells me that we are probably dealing with Desfluran/Desflurane/Desfluoran here, double-checked with its formula CHF2OCHFCF3.

In a couple of european regulatory documents i found this as HFE-236ea2 ([umweltsbundeamt](https://www.umweltbundesamt.de/sites/default/files/medien/2503/dokumente/treibhauspotentiale_ausgewaehlter_verbindungen_und_deren_gemische.pdf), [Proposal for a Regulation of the European Parliament and of the Council on fluorinated greenhouse gases ](https://www.parlament.gv.at/PAKT/EU/XXVII/EU/02/42/EU_24287/imfname_10987241.pdf), [European Environment Agency, Annex II of Regulation (EU) No 517/2014 (not covered by old Regulation (EC) No 842/2006)](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjn9I246t_yAhW2hf0HHaIsC4kQFnoECAQQAQ&url=https%3A%2F%2Fwww.eea.europa.eu%2Fpublications%2Ffluorinated-greenhouse-gases-2020%2Fdownload&usg=AOvVaw3IpI8UZnGsfmsUdVuROZi2)), all quoting a GWP of 989, apparently based on AR4.

However, in a 2016 textbook called [Advances In Atmospheric Chemistry, Volume 1](https://books.google.at/books?id=3wGyDgAAQBAJ&pg=PA333&lpg=PA333&dq=HCFE-236ea2&source=bl&ots=n5XG6al9Gp&sig=ACfU3U0CsWrVRSXOlqePSiYOvZU__qyz_A&hl=en&sa=X&ved=2ahUKEwjn9I246t_yAhW2hf0HHaIsC4kQ6AF6BAgNEAM#v=onepage&q=HCFE-236ea2&f=false) it is called HCFE-236ea2 and the GWP (have not figured out what gwp yet, but probably safe to assume gwp100 now) is 1790.
Looks like that is the same as AR5.

Comment (rgieseke) [reply globalwarmingpotentials.csv:86]: Thanks for digging into that!

If it's not used in SCMs or IAMs i'd consider removing it ...

**Comment by znicholls** [comment]
+1 from me

**Comment by jkikstra** [comment]
yep go ahead, no objection for me to delete this.

**Comment by rgieseke** [comment]
CSV used in #19

**Pull request closed without merging**